sample/ios/iosApp/listitems/PickerListItem.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+import ViewModelsSample
+
+class PickerListItem: UIView {
+    private let pickerItem = UIPickerView()
```

Comment:
Why having an extra UIView layer with constraints that are essentially "match parent"? It would save a UIView instance to subclass UIPickerView directly. Also, tend to avoid constraints in reusable components like this, they are greatly inefficient.